api/app/Policies/TeamPolicy.php

```diff
@@ -66,6 +77,18 @@ public function update(User $user)
      */
     public function viewAnyTeamMembers(User $user)
     {
-        return $user->isAdmin();
+        return $user->isAbleTo("view-any-teamMembers");
+    }
+
+    /**
+     * Determine whether the user can view a specific teams, team members.
+     * Likely to be updated later to allow the team admin and teammates to view their own team
+     *
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewTeamMembers(User $user)
+    {
+        return $user->isAbleTo("view-team-teamMembers");
```

Comment:
Okay, so this is a team-scoped permission (permissions are named like `action-scope-resource`). That means a user only has this permission in the context of specific teams. To check team-scoped permissions we must [add the relevant team](https://laratrust.santigarcor.me/docs/7.x/usage/teams.html#checking-roles-permissions) to the isAbleTo call. In this case, we must get the relevant team from a second argument passed into the function, like this:
```
    public function viewTeamMembers(User $user, Team $model)
    {
        return $user->isAbleTo("view-team-teamMembers", $model);
     }
```

Reply:
Thanks Tristan I added this in as well as a another condition to ensure admins can view still view any team as well. Let me know what you think. [8d731ca](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5899/commits/8d731cab9564810270b7b705c6489b2624391666)